service/controller/v13/templates/cloudformation/guest/instance.go

```diff
@@ -19,25 +22,21 @@ const Instance = `{{define "instance"}}
         Value: {{ .Instance.Cluster.ID }}-master
   DockerVolume:
     Type: AWS::EC2::Volume
-    DependsOn:
-    - {{ .Instance.Master.Instance.ResourceName }}
     Properties:
       Encrypted: true
-      Size: 100
+      Size: 50
       VolumeType: gp2
-      AvailabilityZone: !GetAtt {{ .Instance.Master.Instance.ResourceName }}.AvailabilityZone
+      AvailabilityZone: {{ .Instance.Master.AZ }}
```

Comment:
This is because the master resource now depends on the volumes?

Reply:
Yes, otherwise it has curcular error :(